spec/services/message_employee_matcher_spec.rb

```diff
@@ -30,7 +30,7 @@
           Timecop.freeze(Time.parse("00:00:00 UTC")) do
             days_after_start = 3
             scheduled_message_time = Time.parse("12:00:00 UTC")
-            scheduled_message = create(:scheduled_message, days_after_start: days_after_start, time_of_day: scheduled_message_time)
+            scheduled_message = create(:scheduled_message, days_after_start: days_after_start + 1, time_of_day: scheduled_message_time)
```

Comment:
My brain is also fried 🍳

My understanding is that what the spec is protecting against is users who should receive a message at `2016/1/2 17:00 PT` receiving the message at `2016/1/1 4:00:00 PST` since that is midnight of the next day in UTC and method was previously only comparing minutes and hours.

Prior to this commit, the spec would create a message to go out at `2016/1/1 12:00:00 PST` and confirm that it was not sent at `2016/1/2 00:00:00 UTC`. The message should, however, be sent in that situation (if it has not already) because `2016/1/2 00:00:00 UTC > 2016/1/1 12:00:00 PST`.